cobalt/site/docs/development/setup-linux.md

```diff
@@ -199,3 +201,222 @@ to debug application performance, but also a great way to debug issues and
+
+      - `QA version`: cobalt_evergreen_5.1.2_x64_sbversion-16_qa_compressed_20240629001855.crx
+    - Right click the file and copy file URL
+
+1. Download and unzip the file
+
+    ```sh
+    export LOCAL_CRX_DIR=/tmp/cobalt_dl
+    rm -rf $LOCAL_CRX_DIR
+    mkdir -p $LOCAL_CRX_DIR
+
+    # paste prebuilt library URL and Download it to /tmp
+    # Please update URL according to your need
+    COBALT_CRX_URL=https://github.com/youtube/cobalt/releases/download/25.lts.1/cobalt_evergreen_5.1.2_x64_sbversion-16_qa_compressed_20240629001855.crx
+
+    wget $COBALT_CRX_URL -O $LOCAL_CRX_DIR/cobalt_prebuilt.crx
+
+    # Unzip the downloaded CRX file
+    unzip $LOCAL_CRX_DIR/cobalt_prebuilt.crx -d $LOCAL_CRX_DIR/cobalt_prebuilt
+    ```
+
+1. Copy the files to the appropriate directories
+
+    ```sh
+    cd $COBALT_SRC
+    mkdir -p $EVERGREEN_DIR/install/lib
+    cp -f $LOCAL_CRX_DIR/cobalt_prebuilt/lib/* $EVERGREEN_DIR/
+    cp -f $LOCAL_CRX_DIR/cobalt_prebuilt/lib/* $EVERGREEN_DIR/install/lib
+    cp -f $LOCAL_CRX_DIR/cobalt_prebuilt/manifest.json $EVERGREEN_DIR/
+    cp -rf $LOCAL_CRX_DIR/cobalt_prebuilt/content $EVERGREEN_DIR/
+    ```
+
+## Build executables running on target platform
+
+The Cobalt library is parpared in above steps. Next step is building an
+executable accompany with StarBoard implementation to load the library.
+
+1. Generate output folder for x64
+
+    ```sh
+    # The x64 build type can be any other build type from evergreen
+    # The sb_api_version should be the same as evergreen
+    export TARGET_PLATFORM=linux-x64x11
+    export TARGET_BUILD_TYPE=devel
+    export TARGET_DIR=out/${TARGET_PLATFORM}_${TARGET_BUILD_TYPE}
+    gn gen $TARGET_DIR --args="target_platform=\"$TARGET_PLATFORM\" build_type=\"$TARGET_BUILD_TYPE\" sb_api_version=$SB_API_VER"
+    ```
+
+1. Build executables
+
+    ```sh
+    ninja -C $TARGET_DIR loader_app_install
+    ninja -C $TARGET_DIR native_target/crashpad_handler
+    ninja -C $TARGET_DIR elf_loader_sandbox_install
+    ```
+
+1. Copy necessary artifacts from evergreen folder
+
+    ```sh
+    mkdir -p $TARGET_DIR/content/app/cobalt
+    cp -r $EVERGREEN_DIR/install/lib/ $TARGET_DIR/content/app/cobalt
+    cp -r $EVERGREEN_DIR/content/ $TARGET_DIR/content/app/cobalt
+    cp $EVERGREEN_DIR/manifest.json $TARGET_DIR/
+    cp $TARGET_DIR/native_target/crashpad_handler $TARGET_DIR/
+
+    # The three executables: crashpad_handler, loader_app and elf_loader_sandbox are now placed in $TARGET_DIR/
+    ```
+
+## Running Cobalt with loader_app
+To launch cobalt, simplely execute `loader_app` in command line:
+```sh
+cd $TARGET_DIR
+./loader_app
+```
+The loader_app will find cobalt library from `content/app/cobalt/lib/` and content from `content/app/cobalt/content/`. The library in `content/app/cobalt/lib/` can be compressed or uncompressed version. For Cobalt 2025 certification, compressed version should be used.
+
+# Running Tests
```

Comment:
Should we just point to https://github.com/youtube/cobalt/blob/main/starboard/doc/evergreen/cobalt_evergreen_overview.md#building-and-running-tests for this section?

Reply:
For nplb, I am going to emphasize that the nplb should be built using Cobalt toolchain instead of partner toolchain. An example from building nplb library, copying content folder and launch nplb via elf_loader_sandbox would be helpful.